README.md

```diff
@@ -41,6 +41,12 @@ To do this follow the instructions in the
 
 * **IBM CPLEX** may be installed using `pip install 'qiskit-optimization[cplex]'` to enable the reading of `LP` files.
 
+* **CVXPY** may be installed using command `pip install 'qiskit-optimization[cvx]'` to install the
+  package. CVXPY being installed will enable the usage of the Goemans-Williamson algorithm as an optimizer `GoemansWilliamsonOptimizer`.
+
+* **Matplotlib** may be installed using command `pip install 'qiskit-optimization[matplotlib]'` to install the
+  package. Matplotlib being installed will enable the usage of graph optimization application classes.
```

Comment:
I believe Matplotlib is needed only of you want to 'draw()' the problem/result since its needed by networkx draw method that these use when drawing. The classes can be used otherwise without matplotlib.